utopia-remix/app/services/fgaClient.server.ts

```diff
@@ -87,14 +90,17 @@ function getCredentials(fgaClientMode: 'local' | 'remote'): CredentialsConfig {
 function getFgaClientMode(): 'local_mock' | 'local' | 'remote' {
   if (ServerEnvironment.environment === 'local') {
     if (ServerEnvironment.FGA_API_HOST == '') {
+      // eslint-disable-next-line no-console
```

Comment:
please, let's use `info`, we just added better linting 😅  (I actually fixed it in my own PR)